cfn-resources/README.md

```diff
@@ -1,27 +1,50 @@
 # cfn-resources
 
-## MongoDB Atlas AWS CloudFormation Custom Resource Type
+## MongoDB Atlas AWS CloudFormation Resources
 
 This folder contains the source code for each of the AWS CloudFormation 
-MongoDB Atlas Resource Types.
+MongoDB Atlas Resources. 
+
+Note these are also hosted on AWS CloudFormation Public Registry under Third Party Extensions. 
 
 ### Resource Status Table
 
-| Resource                             | Status                                                 | Examples                                                                                   |
-|--------------------------------------|--------------------------------------------------------|--------------------------------------------------------------------------------------------|
-| project                              | ![Build](https://img.shields.io/badge/Beta-yellow)     | [./project/test](./project/test)                                                           |
```

Comment:
can you confirm that there were indeed 33 resources developed (including Cloud Provider Access)? google sheets tracker appears to have a few extra

Reply:
There are total of 37 resources (excluded the redos). Out of them 33 are published, 1 is not published and 3 are not implemented lack of APIs.